sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/DigitalTwinsLifecycleSamples.cs

```diff
@@ -48,13 +48,13 @@ internal class DigitalTwinsLifecycleSamples
 
         private readonly string _eventRouteId = $"sampleEventRouteId-{Guid.NewGuid()}";
 
-        private readonly string eventhubEndpointName;
-        private readonly DigitalTwinsClient client;
+        private readonly string _eventhubEndpointName;
+        private readonly DigitalTwinsClient _client;
 
         public DigitalTwinsLifecycleSamples(DigitalTwinsClient dtClient, string eventhubEndpointName)
         {
-            this.eventhubEndpointName = eventhubEndpointName;
-            client = dtClient;
+            _eventhubEndpointName = eventhubEndpointName;
+            _client = dtClient;
```

Comment:
We didn't name it this way for the sample snippets. Please revert.

Reply:
I totally forgot! :D